UnitTests/UnitTests.UWP/Extensions/Test_VisualTreeExtensions.cs

```diff
@@ -439,6 +439,103 @@ public async Task Test_VisualTree_FindDescendants_NotFound()
             });
         }
 
+        [TestCategory("VisualTree")]
+        [TestMethod]
+        public async Task Test_VisualTree_FindFirstLevelDescendants_Exists()
+        {
+            await App.DispatcherQueue.EnqueueAsync(async () =>
+            {
+                var treeRoot = XamlReader.Load(@"<Page
+    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""> <!-- Starting Point -->
+    <Grid>
+        <Border x:Name=""A""/>
+        <TextBlock x:Name=""B""/>
+        <StackPanel x:Name=""C"">
+            <TextBox/>
+            <TextBlock/>
+        </StackPanel>
+        <Grid x:Name=""D"">
+            <TextBlock/>
+            <StackPanel>
+                <TextBox/>
+                <TextBlock/>
+            </StackPanel>
+            <TextBlock/>
+        </Grid>
+        <TextBlock x:Name=""E""/>
+    </Grid>
+</Page>") as Page;
+
+                // Test Setup
+                Assert.IsNotNull(treeRoot, "XAML Failed to Load");
+
+                // Initialize Visual Tree
+                await SetTestContentAsync(treeRoot);
+
+                // Main Test
+                var rootGrid = treeRoot.FindDescendant<Grid>();
+                var children = rootGrid.FindFirstLevelDescendants().ToArray();
+
+                Assert.AreEqual(5, children.Length, "Expected to find 5 children.");
+
+                Assert.IsTrue(children.Any(c => ((FrameworkElement)c).Name == "A"), "Couldn't find child 'A'");
+                Assert.IsTrue(children.Any(c => ((FrameworkElement)c).Name == "A"), "Couldn't find child 'B'");
```

Comment:
Need to find child elements B, C, D, and E (copy & paste error?).

Reply:
Whoops, thanks! Fixed in 99f9f4e1f998f9c40cb77c6035ca8c1844128c88 👍